firewood/tests/v2api.rs

```diff
@@ -0,0 +1,69 @@
+use std::{error::Error, path::PathBuf, sync::Arc};
+
+use firewood::{
+    db::{BatchOp, Db as PersistedDb, DbConfig, DbError, WalConfig},
+    v2::api::{Db, DbView, Proposal},
+};
+
+#[tokio::test(flavor = "multi_thread")]
```

Comment:
The test would be simpler if you didn't use the test macro and simply
```rust
fn smoke() -> Result<(), Box<dyn Error>> {
    let cfg = // ...
    let tmpdbpath = tmp_dir().join("testdb");
    let db = PersistedDb::new(tmpdbpath, &cfg).unwrap();
    
    tokio::block_on(async move {
        // then put your test logic here. 
    });
}
```

Then `spawn_blocking` wouldn't be necessary

Reply:
Not important, just a suggestion